workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/AccountBusinessEventsRestController.java

```diff
@@ -158,9 +222,43 @@ private String _getBusinessEvents(JSONArray jsonArray) {
 		return StringUtil.merge(businessEvents, "\n\n");
 	}
 
+	private String _getHighestHeatTag(JSONArray jsonArray) {
+		String highestHeatTag = StringPool.BLANK;
+
+		for (int i = 0; i < jsonArray.length(); i++) {
+			highestHeatTag = _getHighestHeatTag(
+				highestHeatTag, jsonArray.getJSONObject(i));
+		}
+
+		return highestHeatTag;
+	}
+
+	private String _getHighestHeatTag(String heatTag, JSONObject jsonObject) {
```

Comment:
It's be clearer to have this method return the heat tag, and the _getHighestHeatTag do the comparison